docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,404 @@
+---
+title: All about fees
+sidebar_position: 0
+tags: [fees, accounts, transactions, cli, contracts]
+---
+
+The Aztec network is a privacy preserving layer 2 secured by Ethereum, and for the consensus mechanism to work, Aztec makes use of an asset to pay for transactions called, "Fee juice".
```

Comment:
```suggestion
The Aztec network is a privacy preserving layer 2 secured by Ethereum, and for the consensus mechanism to work, Aztec makes use of an asset to pay for transactions called, "Fee juice".  Fee juice is non- transferable on Aztec, which creates some user experience hurdles. To get around this, Aztec has fee abstraction, which means accounts can pay transaction fees on behalf of other accounts. This tutorial will demonstrate some of the ways to take advantage of fee abstraction and pay transaction fees on Aztec (e.g. Fee Payment Contracts).
```

Reply:
I don't want to deter readers in the first paragraph, but I'll amend.